docs/services/featureflagservice.md

```diff
@@ -0,0 +1,110 @@
+# featute flag service
```

Comment:
```suggestion
# Feature Flag Service
```

Reply:
All service docs are lower case.
It also bothers me, but I have just followed the pattern.

Reply:
We seem to have inconsistencies about the capital letters in service names throughout the docs. Maybe this should be fixed with another PR in all places. Anyway, I'd also start with a capital letter.

@julianocosta89 note also the typo.